core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -303,7 +345,6 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
 
       // Invalid jar path will only print the error log, will not add to file server.
       sc.addJar("dummy.jar")
-      sc.addJar("")
```

Comment:
do we change behavior for this?

Reply:
yes , we are creating path `val uri = new Path(path).toUri` to get schema , if we will pass empty string to create path, then it will throw exception

```
Can not create a Path from an empty string
java.lang.IllegalArgumentException: Can not create a Path from an empty string
	at org.apache.hadoop.fs.Path.checkPathArg(Path.java:126)
	at org.apache.hadoop.fs.Path.<init>(Path.java:134)
	at org.apache.spark.SparkContext.addJar(SparkContext.scala:1880)
```

Reply:
We shouldn't change behavior. Can we try-catch the exception and log?

Reply:
now i kept same behaviour just added one more condition (`if (path == null || path.isEmpty)`).